### PR TITLE
Update actions & add go 1.19.x and 1.20.x to go workflow

### PR DESCRIPTION
**Pull request by GiddyGoatGaming**
Updates actions and updates go to 1.19.x

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -7,16 +7,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ 1.17.x, 1.18.x ]
+        go: [ 1.17.x, 1.18.x, 1.19.x ]
```
Comment (tmthrgd) [inline .github/workflows/go.yml:10]: @GiddyGoatGaming Could you remove `1.17.x` at the same time please? We only need to test the two most recent supported versions.

Comment (GiddyGoatGaming) [reply .github/workflows/go.yml]: > @GiddyGoatGaming Could you remove `1.17.x` at the same time please? We only need to test the two most recent supported versions.

Done!

**Review by tmthrgd — APPROVED** [review]
LGTM. (@miekg I haven't checked if there are any important changes between the action versions that might need to be considered).

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -7,16 +7,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ 1.17.x, 1.18.x ]
+        go: [ 1.18.x, 1.19.x ]
```
Comment (tmthrgd) [inline .github/workflows/go.yml:10]: Go1.20 is now out. Can you change this to 1.19.x and 1.20.x?

**Review by tmthrgd — APPROVED** [review]

**Comment by miekg** [comment]
think it's just fine to go to v3

**Pull request merged**